src/Framework/MockObject/MockBuilder.php

```diff
@@ -178,6 +178,31 @@ public function setMethods(array $methods = null): self
         return $this;
     }
 
+    /**
+     * Specifies the subset of methods to mock, requiring each to exist in the class
+     */
+    public function setRealMethods(array $methods = null): self
+    {
+        if ($methods) {
+            $reflection = new \ReflectionClass($this->type);
```

Comment:
Argument 1 of ReflectionClass::__construct expects class-string<T>|object, parent type string provided
```
            $reflection = new \ReflectionClass($this->type);
                                               ^^^^^^^^^^^
```